internal/testutils/coverage.go

```diff
@@ -64,16 +65,20 @@ func TrackTestCoverage(t *testing.T) (testCoverFile string) {
 // MergeCoverages append all coverage files marked for merging to main Go Cover Profile.
 // This has to be called after m.Run() in TestMain so that the main go cover profile is created.
 // This has no action if profiling is not enabled.
-func MergeCoverages() error {
+func MergeCoverages() (err error) {
```

Comment:
without decorate, as the return argument is obvious, I wouldn’t declare `err` here, but just `var err error` below. It’s just that implicit declarations are… hem (but necessary in case of decorate, unfortunately or to give hint to the caller about the input/output when necessary)

Reply:
I'll fix it if we decide to go with this approach, but I doesn't really seem worth to bump the version just for it...